### ISSUE TITLE
Bold text converted into italic in HTML

### ISSUE THREAD

**Issue by againxx** [vimwiki/vimwiki#977]
Prior to submitting a new issue make sure to complete these steps:

- [X] Checkout the `dev` branch and confirm the issue is present there as well.
      The `dev` branch contains fixes that may not have been merged to `master` yet.
- [X] Post the syntax you are using (default/mediawiki/markdown) **and** your vimwiki settings from your `.vimrc`
- [X] Provide a detailed description of the problem including **steps to reproduce the issue**.
- [X] Include the output of `:VimwikiShowVersion`.

After recent update from `dev` branch, the exported html page seems to use `<em>` for bold text surrounded by `*`, which is same as italic.

I am using vimwiki syntax, and these are my vimwiki settings in `.vimrc`
```
let g:vimwiki_global_ext = 0
let g:vimwiki_list = [
\   {
\       'path': '~/Documents/Vimwiki/wiki',
\       'css_name': 'css/style.css',
\       'path_html': '~/Documents/Vimwiki/html',
\       'template_path': '~/Documents/Vimwiki/template',
\       'syntax': 'default',
\       'ext': '.wiki',
\       'links_space_char': '_',
\       'auto_tags': 1,
\       'auto_export': 1,
\       'nested_syntaxes': {'python': 'python', 'cpp': 'cpp'}
\   },
\   {
\       'path': '~/Documents/Notes',
\       'syntax': 'markdown',
\       'ext': '.md',
\       'links_space_char': '_',
\       'auto_tags': 1
\   },
\   {
\       'path': '~/Documents/Todos',
\       'syntax': 'markdown',
\       'ext': '.md',
\       'index': 'todo'
\   }
\ ]

let g:vimwiki_key_mappings = {
\   'table_format': 0,
\   'table_mappings': 0,
\ }
let g:vimwiki_folding = 'expr'
" let g:vimwiki_filetypes = ['markdown']
let g:vimwiki_table_auto_fmt = 0
let g:vimwiki_hl_headers = 1
```

And my `:VimwikiShowVersion` result
```
Os: Linux
Vim: 800
Branch: dev
Revision: aa628f8
Date: 2020-08-07 02:39:53 -0400
```

Another no very relevant issue is that the syntax analysis for italic will be confused when using subscript `_` in tex environment, which will makes the concealing inproperly.

Before concealing:
![vimwiki_math2](https://user-images.githubusercontent.com/22728604/89618973-b2e61e00-d8bf-11ea-9dc2-9a36fd0f0604.png)
After concealing:
![vimwiki_math](https://user-images.githubusercontent.com/22728604/89618732-4cf99680-d8bf-11ea-9b79-6b7e555e4bc0.png)

The highlight group for `_{dW}` is `VimwikiItalic` instead of `VimwikiEqIn`.

**Comment by tinmarino** [comment]
## 2 For your second point (italic in equation)

Fixed: 44b6e30bd

```text
__Single Line:__ with surrounding _italic $ \int x^n + y^n = z^n $ endofit_

__Multiple Line:__ _italic_
$$
\sum_{123} A^b
$$

__Good superscript:__ ^asda^
_italic_
```

->

![Screenshot from 2020-08-07 10-42-17](https://user-images.githubusercontent.com/6123962/89657205-af8c6b80-d89a-11ea-88c1-b4629e64cbd1.png)


More info #709

**Comment by tinmarino** [comment]
## 1 For the first issue (html bad italic)

I copied the wrong regex to html.vim (took those from markdown and not wiki)

If : dfbab8e solves this issue, It would be a nice contribution from you to send a good example : File In.Wiki -> File Out.Html so that we can put it in regression tests
